tensorboard/plugins/hparams/tf_hparams_utils/tf-hparams-utils.ts

```diff
@@ -132,7 +132,9 @@ export function hparamValueByIndex(schema, sessionGroup, hparamIndex) {
 export function metricValueByIndex(schema, sessionGroup, metricIndex) {
   const metricName = schema.metricColumns[metricIndex].metricInfo.name;
   const metricValue = metricValueByName(sessionGroup.metricValues, metricName);
-  return metricValue === undefined ? undefined : metricValue.value;
+  return metricValue === undefined || metricValue.value === 'NaN'
```

Comment:
Filters out values with NaNs so they are not rendered on the scalar plot matrix.